src/main/java/Calculator.java

```diff
@@ -0,0 +1,72 @@
+import java.util.Scanner;
+
+public class Calculator {
+    public void Calculator(){};
+    public int operation(String[] values)throws  Exception{
+        this.validation(values);
+
+        return filtNumberAndOperation(values);
+    }
+
+    public int filtNumberAndOperation(String[] values){
+        int result=0;
+        for(int i=0;i<values.length-2;i+=2){
+            result = classifyIndex(i,result,values);
+        }
+        return result;
+    }
+
+    public int classifyIndex(int i, int result, String[] values){
+        int firstNumber = Integer.parseInt(values[i]);
+        int secondNumber = Integer.parseInt(values[i+2]);
+        String op = values[i+1];
```

Comment:
```suggestion
        int firstNumber = Integer.parseInt(values[i]);
        int secondNumber = Integer.parseInt(values[i+SECOND_NUMBER_INDEX]);
        String op = values[i+OPERATION_NUMBER_INDX];
```
이와 같이 매직넘버는 의미를 표현하는 변수명으로 전역 변수 선언해주세요. 
이 코드를 처음 보는 사람들은 의미를 모를 가능성이 높기에 이해를 돕기 위해 선언해주는 편입니다.